TOOLS/ce/src/main/scala/org.opalj.ce/Comment.scala

```diff
@@ -0,0 +1,144 @@
+    }
+
+    // Tells the Comment object that no more comments will be added. The comment object then begins parsing the comments into advanced objects
+
+    /**
+     * Starts parsing the flags within the comment. Can only be called once.
+     * After it was called, no new comments can be added.
+     */
+    def commitComments(): Unit = {
+        if (!commentsCommitted) {
+            for (line <- commentBuffer) {
+                if (line.trim.startsWith("@label")) {
+                    this.label = line.trim.stripPrefix("@label").trim
+                } else if (line.trim.startsWith("@brief")) {
+                    brief = line.trim.stripPrefix("@brief").trim
+                } else if (line.trim.startsWith("@constraint")) {
+                    constraints.addOne(line.trim.stripPrefix("@constraint").trim)
+                } else if (line.trim.startsWith("@type")) {
+                    datatype = line.trim.stripPrefix("@type").trim
+                } else {
+                    description.addOne(line.trim.stripPrefix("@description").trim)
+                }
+            }
+            commentsCommitted = true
+        }
+    }
+
+    /**
+     * Converts the Comment object into HTML syntax.
+     * Commits the Comments if it has not happened previously.
+     * @return returns the entire object as HTML code ready for insertion into an HTML file.
+     */
+    def toHTML: String = {
+        if (!commentsCommitted) commitComments()
+        var HTMLString = ""
+        if (!this.isEmpty) {
+            if (description.nonEmpty) {
+                HTMLString += "<p><b> Description: </b> <br>"
+                for (line <- description) {
+                    HTMLString += line + "<br>"
+                }
+                HTMLString += "</p>"
+            }
+            if (datatype.nonEmpty) HTMLString += "<p><b> Type: </b>" + datatype + "<br></p>"
+            if (constraints.nonEmpty) {
+                if (datatype.equals("enum")) {
+                    HTMLString += "<p><b> Allowed Values: </b><br>"
+                } else {
+                    HTMLString += "<p><b> Constraints: </b><br>"
+                }
+                for (line <- constraints) {
+                    HTMLString += line + "<br>"
+                }
+                HTMLString += "</p>"
+            }
+        }
+        HTMLString
+    }
+
+    /**
+     * Merges another committed comment into this comment.
+     * @param comment accepts the comment that should be merged into this comment.
+     */
+    def mergeComment(comment: Comment): Unit = {
+        // Make sure every comment is committed
+        this.commitComments()
+        comment.commitComments()
+
+        // Merge comments
+        if (this.label != "" || comment.label != "") {
+            this.label = comment.label + "." + this.label
+        }
+        this.brief = comment.brief + "   " + this.brief
+        this.brief = this.brief.trim
```

Comment:
No. If one of the briefs is empty, line 97 adds a whitespace either in front of or behind one of the briefs. I decided to use it like that because it is more compact and mor reliable than checking four possible state combinations of brief 
State 1: comment.brief is empty and this.brief is not
State 2: this.brief is empty and comment.brief is not
State 3: both are empty
State 4: both are not empty

These two lines satisfy all four states and save a large handling operation